views/includes/entity.html

```diff
@@ -20,7 +20,15 @@
   {% endif %}
 {% endset %}
 
-<h2 class="govuk-heading-l">Overseas entity details</h2>
+{% if pageParams.isRegister %}
```

Comment:
Ok, only allow name edit if registration for now. We need a JIRA ticket for Entity name edit.